[debug] [2024-04-29T15:00:47.466Z] ----------------------------------------------------------------------
[debug] [2024-04-29T15:00:47.468Z] Command:       /home/desa04/.nvm/versions/node/v20.12.2/bin/node /home/desa04/.nvm/versions/node/v20.12.2/bin/firebase init
[debug] [2024-04-29T15:00:47.469Z] CLI Version:   13.7.5
[debug] [2024-04-29T15:00:47.469Z] Platform:      linux
[debug] [2024-04-29T15:00:47.469Z] Node Version:  v20.12.2
[debug] [2024-04-29T15:00:47.469Z] Time:          Mon Apr 29 2024 12:00:47 GMT-0300 (Argentina Standard Time)
[debug] [2024-04-29T15:00:47.469Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-04-29T15:00:47.474Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-04-29T15:00:47.474Z] > authorizing via signed-in user (ignacio21496@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /home/desa04/Desktop/AE-Front-End

[debug] [2024-04-29T15:00:58.654Z] Error: Timed out.
    at Timeout._onTimeout (/home/desa04/.nvm/versions/node/v20.12.2/lib/node_modules/firebase-tools/lib/utils.js:261:49)
    at listOnTimeout (node:internal/timers:573:17)
    at process.processTimers (node:internal/timers:514:7)
[error] 
[error] Error: An unexpected error has occurred.
